libnopegl/src/node_filters.c

```diff
@@ -191,6 +207,71 @@ static const struct node_param filtersaturation_params[] = {
 };
 #undef OFFSET
 
+enum {
+    SELECTOR_COMPONENT_LIGHTNESS,
+    SELECTOR_COMPONENT_CHROMA,
+    SELECTOR_COMPONENT_HUE,
```

Comment:
Can you explicitly set their value since `2` is directly referenced in the shader ?

Reply:
done for all 3 enums, with a comment to explain why